src/commands/lightning/preview/app.ts

```diff
@@ -157,13 +158,18 @@ export default class LightningPreviewApp extends SfCommand<void> {
       return Promise.reject(new Error(messages.getMessage('error.no-project', [(error as Error)?.message ?? ''])));
     }
 
+    logger.debug('Configuring local web server identity');
+    const connection = targetOrg.getConnection(undefined);
+    const username = connection.getUsername() as string;
```

Comment:
The assumption was that the user had already authenticated with the org. If that's not the case, and indeed you're right in `getUsername()` returning undefined, then can we error out of the command? Because without having been already authorized and therefore without user name I cannot make identity data or write the token to server. @maliroteh-sf @khawkins

Reply:
Yep, if there's no org authorized, then this command is effectively useless. I don't know if having a `Flags.requiredOrg` argument does that boundary checking for you. But at least as an added check, we can error out if there's no username.